src/_P019_PCF8574.ino

```diff
@@ -40,7 +40,7 @@ TaskDevicePluginConfigLong settings:
 boolean Plugin_019(byte function, struct EventStruct *event, String& string)
 {
   boolean success = false;
-  static int8_t switchstate[TASKS_MAX];
+  static int switchstate[TASKS_MAX];
```

Comment:
Why changing it to an int?

The build error is about this line in both files (P009 and P019)
`if (state != -1 && state != switchstate[event->TaskIndex])`
`state` is a boolean, not an int and thus will this check always return false.

Reply:
I was trying to solve a cosmetic issue on displaying the status of non connected port.
But after digging I found that current implementation has a bug: 
Assume a user creates a device and selects a port of PCF or MCP that doesn't exist or is not connected. The internal state of that port is -1, but with current implementation it's converted to 0. I was trying to fix the cosmetic thus I changed the data type.

But I will fix the internal logic too.
In the case presented above I will display -1 so user will now that the port is incorrect or disconnected or simply is not found.

Reply:
In my environement it was compiling fine. Strange.

Reply:
The Travis settings are to consider some warnings as error.
And this is certainly one to consider as an error, since it is unexpected behavior.